autoleagueplay/leaderboard/leaderboard.py

```diff
@@ -38,6 +38,8 @@ def generate_leaderboard(working_dir: WorkingDir, run_strategy: RunStrategy, ext
     # Divisions. We only have color palettes configured for a certain number of them, so enforce a limit.
     divisions = Ladder.DIVISION_NAMES[:len(Symbols.palette)]
 
+    extra = (allow_extra and len(new_ladder.bots) > 40)
```

Comment:
What happens when there are 61 bots?

Reply:
Those will be drawn off screen.

When/if that time comes we should make it automatically expand with more columns, or decide to only draw the top bots.